### DISCUSSION TITLE
CTkEntry?

### DISCUSSION THREAD

**Discussion by ptstevens** [TomSchimansky/CustomTkinter#392] [Q&A] [answered]
Hi i need a box where user can enter an API key - always the same length - with 'Normal' tkinter I just used 
```
auth_token_val = tk.StringVar()
api_token = tk.Text(frame3, width=64, height=2)
api_token.pack()
```
This worked well but using CustomTkinter I cant seem to manage and I dont want to use the old tkinter one as it ruins the look of the gui!!! Basically need a box that takes a long string pasted in and wraps over 2 lines

Any suggestions please

**Comment by TomSchimansky** [comment, marked as answer]
There will be a CTkTextbox in the future, but its not ready yet, maybe in a few days...

**Reply by ptstevens** [reply]
Ahhhh Tom!!!!!! I was hoping for a quick fix!!!! Ok fella thanks for the quick response - I'll have to find a workaround.

**Reply by Sfras1** [reply]
Is there a way to use the "wrap" method on CTkEntry? Having text continue horizontally, instead of automatically returning to the next line, is pretty ugly.
